src/Nest/Document/Multiple/Bulk/BulkRequestFormatter.cs

```diff
@@ -42,12 +41,7 @@ public void Serialize(ref JsonWriter writer, IBulkRequest value, IJsonFormatterR
 				if (body == null)
 					continue;
 
-				var bodySerializer = op.Operation == "update" || body is ILazyDocument
-					? requestResponseSerializer
-					: sourceSerializer;
-
-				var bodyBytes = bodySerializer.SerializeToBytes(body, memoryStreamFactory, SerializationFormatting.None);
-				writer.WriteRaw(bodyBytes);
+				SourceWriter.Serialize(ref writer, body, formatterResolver);
```

Comment:
This looks like a behavioural change?

Reply:
Because `SerializationFormatting.None` is not specified explicitly?

Reply:
The behavioural change looks like the `SourceWriter` is now **always** used. 

Before, the `requestResponseSerializer` was used when the operation was an update or the body was an `ILazyDocument`

Reply:
Ahh you are right, will address!